tests/python-pytest/onnx/test_operators.py

```diff
@@ -253,3 +254,16 @@ def test_onnx_export_dropout(tmp_path, dtype, p):
     M = def_model('Dropout', p=p)
     x = mx.nd.array([[3,0.5,-0.5,2,7],[2,-0.4,7,3,0.2]], dtype=dtype)
     op_export_test('Dropout', M, [x], tmp_path)
+
+@pytest.mark.parametrize('dtype', ['float16', 'float32'])
+def test_onnx_export_softmax(tmp_path, dtype):
+    x = mx.nd.random.uniform(0, 1, (2, 3, 4), dtype=dtype)
```

Comment:
sorry about nitpicking: shall we also test high dimensional + axis = middle dim cases?

Reply:
We might also parametrize temperature?

Reply:
added more tests for temperature and axis